cmd/apiserver-watcher/run.go

```diff
@@ -175,6 +176,18 @@ func (h *handler) onSuccess() error {
 	if err := removeVipStateFile(h.vip, "down"); err != nil {
 		return err
 	}
+	ip := net.ParseIP(h.vip)
+	var nlFamily netlink.InetFamily
+	nlFamily = netlink.FAMILY_V4
+	if ip.To4() == nil && ip.To16() != nil {
+		nlFamily = netlink.FAMILY_V6
+	}
+	// bz1930457 delete stale conntrack entries to the apiserver
+	filter := &netlink.ConntrackFilter{}
+	filter.AddIP(netlink.ConntrackNatSrcIP, ip)
```

Comment:
I'm assuming this is the correct conntrack filter - SrcIp does seem odd, but conntrack is odd.

Reply:
yep, this is 
```	
ConntrackNatSrcIP    = ConntrackReplySrcIP // deprecated use instead ConntrackReplySrcIP
```

that is 
```
	ConntrackReplySrcIP                        // --reply-src ip  Reply Source IP
```

so, it filters the packets that are NATed and coming with the srcIP the apiserver IP ... if you see the issue https://bugzilla.redhat.com/show_bug.cgi?id=1925698#c29

> Deleting the stale flows via conntrack on all masters immediately resolved the issue:
> 
> sh-4.4# conntrack -D -r 10.0.0.2
> tcp      6 119 SYN_SENT src=35.191.11.225 dst=10.0.0.2 sport=32768 dport=6443 [UNREPLIED] src=10.0.0.2 dst=35.191.11.225 sport=6443 dport=32768 mark=0 secctx=system_u:object_r:unlabeled_t:s0 use=1
> tcp      6 119 SYN_SENT src=35.191.2.141 dst=10.0.0.2 sport=32768 dport=6443 [UNREPLIED] src=10.0.0.2 dst=35.191.2.141 sport=6443 dport=32768 mark=0 secctx=system_u:object_r:unlabeled_t:s0 use=1
> tcp      6 110 SYN_SENT src=35.191.2.131 dst=10.0.0.2 sport=32768 dport=6443 [UNREPLIED] src=10.0.0.2 dst=35.191.2.131 sport=6443 dport=32768 mark=0 secctx=system_u:object_r:unlabeled_t:s0 use=1
> conntrack v1.4.4 (conntrack-tools): 3 flow entries have been deleted.